deploy/test-environments/fleet_api/src/api/common_api.py

```diff
@@ -287,3 +375,118 @@ def update_package_version(cfg: Munch, package_name: str, package_version: str):
         logger.error(
             f"API call failed, status code {api_ex.status_code}. Response: {api_ex.response_text}",
         )
+
+
+def bulk_upgrade_agents(cfg: Munch, agent_ids: List[str], version: str, source_uri: str) -> str:
+    """
+    Upgrade a list of agents to a specified version using the Kibana API.
+
+    Args:
+        cfg (Munch): Configuration object containing Kibana URL and authentication details.
+        agent_ids (List[str]): List of agent IDs to upgrade.
+        version (str): The version to upgrade to.
+        source_uri (str): The source URI for the agent package.
+
+    Returns:
+        str: The action ID of the upgrade.
+
+    Raises:
+        APICallException: If the API call fails with a non-200 status code.
+    """
+    # pylint: disable=duplicate-code
```

Comment:
maybe we can edit this check configuration to lint longer duplicate code blocks, since most of our duplicates are API calls.